gateway/src/apicast/http_proxy.lua

```diff
@@ -156,6 +157,13 @@ end
 function _M.request(upstream, proxy_uri)
     local uri = upstream.uri
 
+    if not ngx.var.proxy_authorization then
```

Comment:
where does this var come from?

Reply:
My bad, should be http_proxy_authorization. Fixed in the newest patch